source/administration/changelog.md

```diff
@@ -4,6 +4,121 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v4.10
+
+Release date: 2018-05-16
+
+### Highlights
+
+#### Convert Public Channels to Private
+ - Team and System Admins can now convert a channel to private from the user interface. System Admins can also convert channels back to public [via the commandline](https://docs.mattermost.com/administration/command-line-tools.html#platform-channel-modify).
+
+#### Performance Improvements
+ - Decreased loading time by up to 90% for users with lots of direct and group message channels.
+
+#### Environment Variables Support in GitLab Omnibus
+ - Simplified Mattermost administration by supporting environment variables in GitLab Omnibus.
+ - See [documentation](https://docs.gitlab.com/omnibus/gitlab-mattermost/#upgrading-gitlab-mattermost-from-versions-prior-to-11-0) to learn more.
+
+### Improvements
+
+#### Web User Interface
+ - Added a setting that automatically answers messages sent to a user who is away.
+ - Added a loader for channel invite modal.
+ - Made Shift+up switch keyboard focus to RHS if it's already open to the current thread.
+ - Removed WebRTC end user setting.
+ - Removed support for transparent team icons and added ability to remove team icons.
+ - Added an on-hover effect for image link previews.
+
+#### Performance
+ - Added structured logging.
+ 
+ #### Plugins
+ - Added better plugin error handling and reporting.
+ 
+ #### Slash Commands
+ - Added ``/invite`` slash command to invite users to a channel.
+ - Added ``/platform`` list of commands to teams command in CLI.
+ - Improved slash command error message when payload is invalid JSON.
+ 
+ #### Other (XXXX/// Need help with a category)
+ - Removed ``mm_config`` and ``mm_license`` global state from webapp.
+ - Added support to REST API for sending ephemeral messages to users.
+ - Investigated avoiding duplicating unique indexes.
+ - Added a CLI Command to "Reset to Default" for permission system.
```

Comment:
Section for "Command Line Interface (CLI)".

Also, what's the command? Or link ticket/PR and I can check

Reply:
https://mattermost.atlassian.net/browse/MM-10121

Reply:
@amyblais Sounds like this is permissions related which ships in v5.0. Can you touch base with Eric and George in permissions channel if we should note anything in v4.10 or just wait for 5.0?